tema1 saptamana 3/tema1 saptamana 3/Program.cs

```diff
@@ -0,0 +1,285 @@
+    {
+        odd[q] = array2[o];
+        q++;
+    }
+}
+Console.Write("\nThe Even elements are : \n");
+for (int r = 0; r < p; r++)
+{
+    Console.Write("{0} ", even1[r]);
+}
+Console.Write("\nThe Odd elements are :\n");
+for (int s = 0; s < q; s++)
+{
+    Console.Write("{0} ", odd[s]);
+}
+
+//<<<<<<<<Exercise 6>>>>>>>>>
+Console.WriteLine("Exercise 6");
+Console.WriteLine("Enter the first number");
+int number1=Convert.ToInt32(Console.ReadLine());
+Console.WriteLine("Enter the second number");
+int number2= Convert.ToInt32(Console.ReadLine());
+List<int> numbers=new List<int>();
+if (number1 < number2)
+{
+    for (int t = number1; t <= number2; t++)
+    {
+        numbers.Add(t);
+    }
+}
+else if (number1 > number2)
+{
+    for (int u = number1; u >= number2; u--)
+    {
+        numbers.Add(u);
+    }
+}
+Console.WriteLine("The collection elements: ");
+foreach (int inum in numbers)
+{ Console.WriteLine(inum); }
+Console.WriteLine("the collection elements that divide by 3 : ");
+for(int v=0; v<=numbers.Count; v++)
+{
+    if (numbers[v]%3 == 0)
+    { 
+            Console.WriteLine(numbers[v]);
+    } 
+}
+
+//<<<<<<<<Exercise 7>>>>>>>>>
+Console.WriteLine("Exercise 7");
+List<int> newnumber1 = new List<int>();
+for (int x = 123; x <= 243; x++)
+{
+    newnumber1.Add(x);
+}
+foreach (int no in newnumber1)
+{
+    if ((no % 3 == 0)& (no % 5 == 0))
+    {
+        Console.WriteLine(no + "FizzBuzz");
+    }
+    else if (no % 3 == 0)
+    {
+        Console.WriteLine(no + "Fizz");
+    }
+    else if (no % 5 == 0) 
+    {
+        Console.WriteLine(no + "Bizz");
+    }
+    else 
+    {
+        Console.WriteLine(no);
+    }
+}
+
+//<<<<<<<<Exercise 8>>>>>>>>>
+Console.WriteLine("Exercise 8");
+List<string> Names = new List<string>();
+Console.WriteLine("enter ten names");
+for (int ii = 0; ii < 10; ii++)
+{
+    string input = Console.ReadLine();
+    Names.Add(input);
+}
+int rr = 0;
+while (rr<= 5)
+{
+    Console.WriteLine(Names[rr]);
+    rr++;
+}
+
+//<<<<<<<<Exercise 9>>>>>>>>>
+Console.WriteLine("Exercise 9");
+int inputnumber;
+do
+{
+    inputnumber = Convert.ToInt32(Console.ReadLine());
+}
+while (inputnumber != 0);
+
+//<<<<<<<<Exercise 10>>>>>>>>>
+Console.WriteLine("Exercise 10");
+List<int> newnum = new List<int>();
+int ix = -1; ;
+    do
+{
+    newnum.Add(ix);
+    ix--;
+    int div = newnum[ix] % 21;
+    Console.WriteLine(div);
+    if (div == 0)
+            break;
+}
+    while (ix >= -123);
+foreach (int num in newnum)
+{
+    Console.WriteLine(num);
+}
+
+//<<<<<<<<Exercise 11>>>>>>>>>
+Console.WriteLine("Exercise 11");
+List<string> NamesN = new List<string>();
+Console.WriteLine("Enter ten names");
+/*int i = 0;
+while(i < 10); 
+{
+        string inputi = Console.ReadLine();
+    NamesN.Add(inputi);
+    i++;
+}*/
+
+for(int ia = 0; ia < 10; ia++)
+{
+    string inputi = Console.ReadLine();
+    NamesN.Add(inputi);
+}
+for (int ef = 0; ef < NamesN.Count; ef++)
+{
+    if (NamesN[ef].StartsWith("a"))
```

Comment:
These 4 ifs could be merged into one looking similar to `if(NamesN[ef].StartsWith("a") || NamesN[ef].StartsWith("A") || NamesN[ef].StartsWith("b") || NamesN[ef].StartsWith("B"))`